.travis.yml

```diff
@@ -12,7 +12,7 @@ before_install:
 
 install:
     - rvm install 2.2.2
-    - npm install
+    - yarn
```

Comment:
Can we do `yarn --frozen-lockfile` here? It will fail the build if you update package.json but don't update yarn.lock.